subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationTestReportExtension.kt

```diff
@@ -24,4 +25,12 @@ public class InstrumentationTestReportExtension {
         reportViewer.validate()
         this.reportViewer = reportViewer
     }
+
+    public fun reportViewer(closure: Closure<ReportViewer>) {
```

Comment:
Action must be enough for both groovy and kotlin

Reply:
But it's not working in nested one's; probably we doing something wrong